nbextensions/viewCell/widgets/codeCellRunWidget.js

```diff
@@ -1,170 +0,0 @@
-/*global define*/
```

Comment:
Was this all just dead code?

Reply:
These are directives to "jslint", which was the old linter we used, now replaced by eslint, which has a different way of configuring lint options. So in this case the eslint directive "env.amd = true" takes care of this as well as other globals.

We should actually make sure we are all on the same page with this. I use the config file .eslintrc.yml in the project root which integrates as well with Visual Studio Code, but don't know how other folks set this up.

Reply:
I agree about separating code cleanup from functional changes, although it can be tricky to separate them because the cleanup scope expands as the code change scope expands. Still, an initial crack at it would reduce the noise.

Perhaps an initial code cleanup commit which absorbs the bulk of at least the mechanical fixes (e.g. fixing all linting errors, which may be functional as well, but not part of the main logic changes.)